tests/flow/test_query_validation.py

```diff
@@ -609,3 +609,26 @@ def test40_parser_errors_in_star_projections(self):
             self.env.assertTrue(False)
         except redis.exceptions.ResponseError:
             pass
+
+        # check that AST validation errors are handled correctly
+        # in queries containing star projections
+        try:
+            query = """WITH 1 RETURN *"""
+            redis_graph.query(query)
+            self.env.assertTrue(False)
+        except redis.exceptions.ResponseError:
+            pass
+
+        try:
+            query = """RETURN *"""
+            redis_graph.query(query)
+            self.env.assertTrue(False)
+        except redis.exceptions.ResponseError:
+            pass
+
+        try:
+            query = """CREATE () RETURN DISTINCT *"""
+            redis_graph.query(query)
+            self.env.assertTrue(False)
+        except redis.exceptions.ResponseError:
+            pass
```

Comment:
Can this be extended for `WITH *` scenarios ?
for example:
`MATCH () WITH * RETURN z`
`MATCH () WITH * RETURN *`
`MATCH () WITH * WHERE n.v > 1 RETURN *`

Reply:
Also if you think the following is preferable:
```python
queries = ["WITH 1 RETURN *", "RETURN *", "CREATE () RETURN DISTINCT *"]
for q in queries:
    try:
            redis_graph.query(q)
            self.env.assertTrue(False)
    except redis.exceptions.ResponseError:
            pass
```